chunky/src/java/se/llbit/chunky/renderer/scene/Camera.java

```diff
@@ -788,7 +789,10 @@ public ApertureShape getApertureShape() {
 
   public void setCameraLocked(boolean value) {
     lockCamera = value;
-    scene.refresh();
+    if (scene.mode == RenderMode.PREVIEW) {
```

Comment:
Why would we need to refresh the scene at all just because the camera is locked/unlocked?